vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/overview/ProjectOverviewPage.java

```diff
@@ -28,11 +32,14 @@ public class ProjectOverviewPage extends Div {
 
   private static final Logger log = LoggerFactory.logger(ProjectOverviewPage.class);
 
-  public ProjectOverviewPage(@Autowired ProjectOverviewComponent projectOverviewComponent) {
-    add(projectOverviewComponent);
+  public ProjectOverviewPage(@Autowired ProjectInformationService projectInformationService,
+      ClientDetailsProvider clientDetailsProvider) {
```

Comment:
Basically what i would suggest is: 

1.) Either Introduce a dedicated Div "ProjectOverviewContent" which will be autowired here and just contain the ProjectCollection or 

2.) We skip this for now and argue that the collection is the "content" section of our homepage since we don't have multiple components for now. 

I'm curious why we don't autowire the projectCollection here?